ansible/roles/test/tasks/vlantb.yml

```diff
@@ -0,0 +1,20 @@
+#-----------------------------------------
+# Apply Vlan configuration
+#-----------------------------------------
+- name: Vlan test setup on testbed
+  include: vlan_configure.yml
+  tags: vlan_configure
+
+#-----------------------------------------
+# Run Vlan test
+#-----------------------------------------
+- name: Vlan test run on testbed
+  include: vlan_test.yml
+  tags: vlan_test
+
+#-----------------------------------------
+# Clean up Vlan configuration
+#-----------------------------------------
+- name: Clean up Vlan test configuration on the testbed
+  include: vlan_cleanup.yml
+  tags: vlan_cleanup
```

Comment:
suggest adding using log analyzer to analyze the errors in the log as part of the test as ACL test and Everflow test did, by this could make these test cases more complete.

Reply:
Thank you kebo, LogAnalyzer will make the test case more complete. But I think it's better for me to add LogAnalyzer later after the test check in due to the time factor.